src/requestLogger.js

```diff
@@ -64,6 +64,7 @@ class RequestLogger {
         invocationId: this.invocationId,
         message: {
           title: 'Payload too large',
+          level: payload && payload.message && payload.message.level,
```

Comment:
I would make this `Error` as in this message we are logging is not the log message, but a message that the log message is flawed.

Reply:
What about this?

```
level: 'ERROR',
originalLevel: payload && payload.message && payload.message.level
```

Reply:
So what I didn't share is that this library doesn't know anything about what a payload looks like, but doing this we are guessing that there is both:
* A `level` field in existence, the `level` field is in this place
* All other properties are not important and `level` is the only one.

The real problem is that the original log message is flawed, and while we can make some changes here, at the end of the day the only solution is to change the log messages so they don't get truncated. Because a truncated log message depending on your system may just end up worthless in your logging/monitor/alerting system because it won't be JSON. :(